quantize.c

```diff
@@ -110,77 +85,72 @@ void checkpoint_init_weights(TransformerWeights *w, Config* p, float* f, int sha
 
     w->w1 = ptr;
     ptr += p->n_layers * p->dim * p->hidden_dim;
-
     w->w2 = ptr;
     ptr += p->n_layers * p->hidden_dim * p->dim;
-
     w->w3 = ptr;
     ptr += p->n_layers * p->dim * p->hidden_dim;
 
     w->rms_final_weight = ptr;
     ptr += p->dim;
 
     w->freq_cis_real = ptr;
-    int head_size = p->dim / p->n_heads;
     ptr += p->seq_len * head_size / 2;
-
     w->freq_cis_imag = ptr;
     ptr += p->seq_len * head_size / 2;
+
     w->wcls = shared_weights ? w->token_embedding_table : ptr;
 }
 
+void get_max_vals(float *ptr, int size, float* pmin, float* pmax){
```

Comment:
we would be better off updating the name from `get_max_vals` to `get_minmax` ?

Reply:
done